src/wasmv1_execution/abi/abis.rs

```diff
@@ -235,6 +239,21 @@ pub(crate) fn abi_call(
     )
 }
 
+fn native_address_to_string(
+    handler: &mut super::handler::ABIHandler<'_, '_>,
+    address: NativeAddress,
+) -> Result<String, WasmV1Error> {
+    let address = handler
+        .interface
+        .native_address_to_string(
```

Comment:
In these ABI, you seem to call methods on the interface that I can't find in massa. Did I misunderstand something or what branch of massa should I go to?

Reply:
You're correct
It's a method I'll have to implement.
I haven't found any better way to do this. 
From my point of view the conversion has to be done in massa/models/address
what's your thought?

Reply:
Yes it makes sense, this way we don't forget to update them if we change the address format for example. It was just to make sure ;)